doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -177,6 +177,8 @@ Serial LTE modem
   * The :ref:`CONFIG_SLM_CUSTOMER_VERSION <CONFIG_SLM_CUSTOMER_VERSION>` Kconfig option for customers to define their own version string after customization.
   * The optional ``path`` parameter to the ``#XCARRIEREVT`` AT notification.
   * ``#XCARRIERCFG`` AT command to configure the LwM2M carrier library using the LwM2M carrier settings (see the :kconfig:option:`CONFIG_LWM2M_CARRIER_SETTINGS` Kconfig option).
+  * Option to set the ``PEER_VERIFY`` socket option for ``#XTCPCLI``, ``#XUDPCLI`` and ``#XHTTPCCON``. By default set to ``TLS_PEER_VERIFY_REQUIRED``.
+  * Option to set the ``TLS_HOSTNAME`` socket option to ``null`` to disable the hostname verification for ``#XTCPCLI``, ``#XUDPCLI`` and ``#XHTTPCCON``.
```

Comment:
I'm about 50-50 on if this should be in the Updated section instead. If these options being added is more important, they can stay where they are with the following modifications:
```suggestion
  * Option to set the ``PEER_VERIFY`` socket option for the ``#XTCPCLI``, ``#XUDPCLI`` and ``#XHTTPCCON`` AT commands.
    Set to ``TLS_PEER_VERIFY_REQUIRED`` by default.
  * Option to set the ``TLS_HOSTNAME`` socket option to ``NULL`` to disable the hostname verification for the ``#XTCPCLI``, ``#XUDPCLI`` and ``#XHTTPCCON`` AT commands.
```
If the commands that were updated are more important, then this should be under Updated:
```suggestion
  * The ``#XTCPCLI``, ``#XUDPCLI`` and ``#XHTTPCCON`` AT commands with options to:
  
    * Set the ``PEER_VERIFY`` socket option.
      Set to ``TLS_PEER_VERIFY_REQUIRED`` by default.
    * Set the ``TLS_HOSTNAME`` socket option to ``NULL`` to disable the hostname verification.
```
Could also be split so that one entry is under Added, and the other is under Updated.

Reply:
Chose the update-option. It is unlikely that someone will be affected by this change, but in case this happens, this is more easy to notice from updated side.

Reply:
Thanks for providing options!

Reply:
No problem!